doc/content/examples/example6a.md

```diff
@@ -0,0 +1,65 @@
+
+
+## Modeling in MASTODON
+
+!listing examples/ex06a/psf_grav.i
+
+The mesh consisting of the solid block and the soil is generated by importing an existing mesh file [fig:ex06input].
+The details about the type of mesh file supported can be found at [GettingStarted](manuals/user/index.md). The displacements are defined in the variables section and the velocity, acceleration, normal forces and the tangential (frictional) forces are defined as the auxiliary variables.
+
+!media media/examples/ex06/ex06_input.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:ex06input
+       caption=Input model in MASTODON.
+
+
+[TensorMechanics](syntax/index.md) and [InertialForce](syntax/index.md) kernels are used to model the dynamics of the 3D solid elements. [Gravity](syntax/index.md) kernels is used to apply the gravitational force. The [Newmark time integration](manuals/theory/index.md) parameters used in this problem correspond to the Newmark's average acceleration method, i.e. `beta = 0.25` and `gamma = 0.5` and damping is ignored.
+
+The [ComputeIsotropicElasticityTensorBeam](syntax/index.md) block is used to create the elasticity tensor of the solid block and the elastic soil, using the parameters, Young's Modulus and Poisson's ratio. The stresses and strain is calculated using [ComputeFiniteStrainElasticStress](syntax/index.md) and [ComputeFiniteStrain](syntax/index.md). The densities are assigned to the solid block and the soil using [GenericConstantMaterial](syntax/index.md).
+
+The bottom and the sides of the soil are fixed. A uniform vertical pressure of 5 psi is applied at the top of the solid block and the block is displaced by the prescribed amount as shown in [input_motion].
+
+!plot scatter filename=examples/ex06a/input_motion.csv
+              data=[{'x':'time', 'y':'displacement'}]
+              layout={'xaxis':{'title':'Time (secs)'},'yaxis':{'title':'Displacement (in)'}, 'title':'Displacement of the block'}
+              caption=Input displacement of the block
+              id=input_motion
+
+[Contact](manuals/user/index.md) is implemented using the Contact block. The slave and the master surfaces are defined in the contact block to implement slave-master concept for the contact problem which prevents the nodes of the slave boundary to penetrate the surface elements of the master boundary. Columb friction model with a friction coefficient of 0.7 is used. The penalty method with a penalty parameter of 1e5 is used to solve the contact problem. The penalty method allows small penetration and the contact force is proportional to it. Large penalty allows small penetration.
```

Comment:
...is defined in MASTODON using the 'Contact' block. This model is simulating Coulomb friction, with a coefficient of friction value equal to 0.7. The formulation implemented and demonstrated in this example uses a master/slave concept in which the nodes of the slave boundary are restricted by the nodes of the master surface boundary. Penetration is enforced using a penalty factor which is inversely related to the amount of penetration allowed (large penalty factor results in small amount of penetration and vice versa). The factor used in this example is 1e5.